src/modules/Rentable/index.ts

```diff
@@ -2,19 +2,27 @@ import { SDK } from "../../sdk";
 import { ethers, BigNumber, Contract } from "ethers";
 import DCNT4907A from '../../contracts/DCNT4907A.json';
 import { MetadataRendererInit } from '../MetadataRenderer';
+import { TokenGateConfig } from '../Edition';
 
 const deploy = async (
   sdk: SDK,
   name: string,
   symbol: string,
+  adjustableCap: boolean,
```

Comment:
flagging name is misleading would be better as `isAdjustableCap`
```suggestion
  isAdjustableCap: boolean,
```